### PR TITLE
Add Hong Kong Cantonese localization...

### PR DESCRIPTION
**Pull request by Edditoria**
The translation is completed corresponding to the commit fa0ded9 .

You may see that there are some English words in it, but don't worry.
This is Hong Kong culture. Please see the README in dev repo for details:

https://github.com/Edditoria/notepad-plus-plus-localization-hong-kong

### CODE REVIEW

**Comment by Edditoria** [comment]
Sorry I'm not familiar with C++. I'm afraid I will break your build if I change other files.

But I ticked "Allow edits from maintainers" in this PR, so you can continue to add more things. Thank you.

**Comment by sasumner** [comment]
>Sorry I'm not familiar with C++. I'm afraid I will break your build if I change other files.

Adding a new "localization" doesn't require C++ knowledge.
You shouldn't change any other files.

**Comment by donho** [comment]
>> Sorry I'm not familiar with C++. I'm afraid I will break your build if I change other files.

> Adding a new "localization" doesn't require C++ knowledge.
> You shouldn't change any other files.

No worry, I'll be in charge of the rest stuffs.

File: PowerEditor/installer/nativeLang/hongKongCantonese.xml [hunk 1]
```diff
@@ -0,0 +1,1456 @@
...
+            <replace-in-files-confirm-filetype value="檔案類型："/>
+            <replace-in-open-docs-confirm-title value="係咪真係先？"/>
+            <replace-in-open-docs-confirm-message value="係咪真係要响所有打開咗嘅文件度執行取代？"/>
+            <find-result-caption value="搵到嘅結果"/>
+            <find-result-title value="搵"/>
+            <find-result-title-info value="（搵到 $INT_REPLACE1$ 個結果，散佈响 $INT_REPLACE2$ 份文件，總共搵咗 $INT_REPLACE3$ 份文件）"/>
+            <find-result-title-info-selections value="（搵到 $INT_REPLACE1$ 個結果，散佈响 $INT_REPLACE2$ 個揀選範圍，總共搵咗 $INT_REPLACE3$ 份文件）"/>
+            <find-result-title-info-extra value=" - 搵中搵模式：只顯示「响搜尋結果再搵」嘅篩選結果"/>
+            <find-result-hits value="（搵到 $INT_REPLACE$ 個結果）"/>
+            <find-regex-zero-length-match value="我覺得自己係零囉" /> <!-- 字元相符不過長度係零 -->
```
Comment (donho) [inline PowerEditor/installer/nativeLang/hongKongCantonese.xml:1453]: LOL

**Pull request closed without merging**